tests/drivers/pwm/pwm_api/src/test_pwm.c

```diff
@@ -38,6 +38,10 @@
 #define PWM_DEV_NAME DT_LABEL(DT_ALIAS(pwm_2))
 #elif DT_NODE_HAS_STATUS(DT_ALIAS(pwm_3), okay)
 #define PWM_DEV_NAME DT_LABEL(DT_ALIAS(pwm_3))
+
```

Comment:
nit: drop the extra blank line on lines 41 and 44.

Reply:
I put them on purpose, to clearly separate 'alias' way of working and 'dts compatible' way of working.